book/02-git-basics/sections/remotes.asc

```diff
@@ -1,18 +1,14 @@
 [[r_remote_repos]]
-=== Working with Remotes
+=== Trabalhando de Forma Remota
 
-To be able to collaborate on any Git project, you need to know how to manage your remote repositories.
-Remote repositories are versions of your project that are hosted on the Internet or network somewhere.
-You can have several of them, each of which generally is either read-only or read/write for you.
-Collaborating with others involves managing these remote repositories and pushing and pulling data to and from them when you need to share work.
-Managing remote repositories includes knowing how to add remote repositories, remove remotes that are no longer valid, manage various remote branches and define them as being tracked or not, and more.
-In this section, we'll cover some of these remote-management skills.
+Para colaborar com qualquer projeto Git, você precisará saber como gerenciar seus repositórios remotos. Repositórios remotos são versões de seu repositório hospedado na Internet ou em uma rede qualquer.
+Você pode ter vários deles, cada um dos quais geralmente é ou somente leitura ou leitura/escrita.
+Colaborar com outras pessoas envolve o gerenciamento destes repositórios remotos e <<TRANSLATION_NOTES.asc#atualizar, atualizar>>(_pushing_) e <<TRANSLATION_NOTES.asc#obter,obter>>(_pulling_) dados para e deles quando você precisar compartilhar seu trabalho.
```

Comment:
You can not make references to `TRANSLATION_NOTES.asc` because the file is not included in the book.

Reply:
Ok! Thank you! 😊

Reply:
What you can do is to add an include line to `TRANSLATION_NOTES.asc` at the end of `progit.asc`, format `TRANSLATION_NOTES.asc` as a first level chapter (using a main `==` title with an anchor named *TRANSLATION_NOTES* like this:

```
[#TRANSLATION_NOTES]
== Notas de Tradução
```

and be careful that the targets of the links that you create to point into this file actually exist. For instance, there is no anchor *`obter`* or *`atualizar`* in `TRANSLATION_NOTES.asc` at the moment, so this would still fail.

Reply:
Ok! I'll try what you said. But I don't understand what happened. I removed the references and cleaned the code....

Reply:
I tried what you said and I created the anchors on files. Testing here, using the Firefox add-on AsciiDoctor, I could see that the references are almost ok... When I click on the link page to access the `TRANSLATION_NOTES.asc`, I receive the error "File Not Found". Seeing the URI, I can see that the link really points to reference file:
`file:///C:/Users/vanes/OneDrive/Projetos/OpenSource/progit/progit2-pt-br/TRANSLATION_NOTES.adoc#pushing_definition`.